webpack/scenes/ContentViews/Details/Versions/Delete/RemoveSteps/CVReassignHostsForm.js

```diff
@@ -122,12 +153,24 @@ const CVReassignHostsForm = () => {
           <p>{multiCVRemovalInfo}</p>
         </Alert>
       )}
+      {showAlert &&
+        <Alert
+          variant="info"
+          ouiaId="disabled-environments-alert"
+          isInline
+          title={__('Some environments are disabled because they are not associated with the host\'s content source.')}
```

Comment:
```suggestion
          title={__('Some environments are disabled because they are not associated with all of the affected hosts\' content sources.')}
```